tests/conftest.py

```diff
@@ -177,18 +177,19 @@ def jaxsim_model_sphere() -> js.model.JaxSimModel:
 
 
 @pytest.fixture(scope="session")
```

Comment:
Is this still necessary now?
```suggestion
```

Reply:
This can be computed only once and cached for the entire pytest session. It is not necessary but why not caching it.

Reply:
Alright, thanks for the explanation